app/src/main/java/com/github/polybooks/FilteringBooksActivity.kt

```diff
@@ -110,26 +110,26 @@ class FilteringBooksActivity : AppCompatActivity() {
     }
 
     private fun getOrdering() : BookOrdering {
-        if(mSortPopularity.isChecked) return BookOrdering.DEFAULT //add it when available
-        if(mSortTitleDec.isChecked) return BookOrdering.TITLE_DEC //add it when available
-        if(mSortTitleInc.isChecked) return BookOrdering.TITLE_INC //add it when available
+        if (mSortPopularity.isChecked) return BookOrdering.DEFAULT //add it when available
+        if (mSortTitleDec.isChecked) return BookOrdering.TITLE_DEC //add it when available
+        if (mSortTitleInc.isChecked) return BookOrdering.TITLE_INC //add it when available
         else return BookOrdering.DEFAULT
     }
 
     private fun getInterests() : Set<Interest> {
         val interests = mutableSetOf<Interest>()
-        if(mFieldCS.isChecked) interests.add(Field(mFieldCS.text.toString()))
-        if(mFieldBio.isChecked) interests.add(Field(mFieldBio.text.toString()))
-        if(mFieldArchi.isChecked) interests.add(Field(mFieldArchi.text.toString()))
-
-        if(mCourseCOM480.isChecked) interests.add(Course(mCourseCOM480.text.toString()))
-        if(mCourseCS306.isChecked) interests.add(Course(mCourseCS306.text.toString()))
-
-        if(mSemBa1.isChecked) interests.add(Semester(mSemBa1.text.toString(), "?"))
-        if(mSemBa2.isChecked) interests.add(Semester(mSemBa2.text.toString(), "?"))
-        if(mSemBa3.isChecked) interests.add(Semester(mSemBa3.text.toString(), "?"))
-        if(mSemMa1.isChecked) interests.add(Semester(mSemMa1.text.toString(), "?"))
-        if(mSemMa2.isChecked) interests.add(Semester(mSemMa2.text.toString(), "?"))
+        if (mFieldCS.isChecked) interests.add(Field(mFieldCS.text.toString()))
+        if (mFieldBio.isChecked) interests.add(Field(mFieldBio.text.toString()))
+        if (mFieldArchi.isChecked) interests.add(Field(mFieldArchi.text.toString()))
+
+        if (mCourseCOM480.isChecked) interests.add(Course(mCourseCOM480.text.toString()))
+        if (mCourseCS306.isChecked) interests.add(Course(mCourseCS306.text.toString()))
+
+        if (mSemBa1.isChecked) interests.add(Semester(mSemBa1.text.toString(), "?"))
+        if (mSemBa2.isChecked) interests.add(Semester(mSemBa2.text.toString(), "?"))
+        if (mSemBa3.isChecked) interests.add(Semester(mSemBa3.text.toString(), "?"))
+        if (mSemMa1.isChecked) interests.add(Semester(mSemMa1.text.toString(), "?"))
+        if (mSemMa2.isChecked) interests.add(Semester(mSemMa2.text.toString(), "?"))
 
```

Comment:
Maybe a refactorization of this function to avoid duplication is possible ?

Reply:
I think those appear because he merged the content one of my previous pr. I didn't bother with duplication because those hardcoded parameters are not meant to last :)

Reply:
Yeah correct I merged the main branch into mine (because of the sale/book changes in the database, I needed the most recent version)

@so7fie yeah I assumed you forgot it ^^'. And I also did it for Joshua (I merge your + Josh's work in my PR). There were those spaces missing, commented previous lines of code that should have been deleted, sometimes two tabs instead of one, sometimes spaces before `:`, sometimes after and sometimes both on 3 contiguous lines of code, etc. I think we should all try to make the code more clear whenever we see fit as Mr. Candea said during the lecture on refactoring last semester ^^. 

No but more seriously, we should have defined coding guidelines for a project of this scale at the very beginning of the project imo... because the different parts of the code seem completely heterogeneous

Reply:
@Laniakea1999 Tip: you can review commit by commit (this is personnally what I do). This way, you can "skip" commits that are not created by the PR owner (and it saves you a bunch of time x) )

Reply:
@so7fie I let you decide whether you want to refactorise as Laniakea said then ^^'

Reply:
I will probably do it in another pr, so let's ignore the duplications for this one